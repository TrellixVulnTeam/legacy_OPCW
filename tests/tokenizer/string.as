"Hello World"
"Hi \x32 \n\?"
'SH'
"'"
"\""
'\''
'"'
"'"
