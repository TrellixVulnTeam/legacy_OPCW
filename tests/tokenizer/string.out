<string> 'Hello World'
<string> 'Hi \x32 \n\?'
<string> 'SH'
<string> '''
<string> '\"'
<string> '\''
<string> '"'
<string> '''
<end>
